_posts/2022-03-10-Socratic-Seminar-02.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: post
+type: socratic
+title: "Socratic Seminar 02"
+meetup: https://www.meetup.com/BitDevsNYC/events/283380081/
+---
+
+
+# Conversation
```

Comment:
1. [Feature Request: BIP47 payment codes for private donations](https://github.com/bitcoindevkit/bdk/issues/549)
    - [What are PayNyms?](https://bitcoiner.guide/paynym/#:~:text=The%20structure%20of%20BIP47%20dictates,of%20any%20PayNym%20generated%20transactions.)
    - [BIP 0047](https://github.com/bitcoin/bips/blob/master/bip-0047.mediawiki)

2. New Bitcoin scaling whitepaper just dropped: ["CoinPool: efficient off-chain payment pools for Bitcoin"](https://coinpool.dev/v0.1.pdf)